package.json

```diff
@@ -97,7 +97,8 @@
 						"off"
 					],
 					"default": "info",
-					"description": "%githubPullRequests.logLevel.description%"
+					"description": "%githubPullRequests.logLevel.description%",
+					"deprecationMessage": "%githubPullRequests.logLevel.deprecationMessage%"
```

Comment:
Probably use `markdownDeprecationMessage` so that the set log level command can be rendered as link?

Reply:
Good point. Opened https://github.com/microsoft/vscode-pull-request-github/pull/4494